packages/seo-plugin/src/seo-interface/components/FocusKeyphrase.vue

```diff
@@ -0,0 +1,141 @@
+<script setup lang="ts">
+import { onClickOutside, onKeyStroke } from '@vueuse/core';
+import { computed, ref } from 'vue';
+
+const props = defineProps<{
+	modelValue: string;
+	disabled?: boolean;
+}>();
+
+const emit = defineEmits<{
+	'update:modelValue': [value: string];
+}>();
+
+const value = computed({
+	get: () => props.modelValue || '',
+	set: (val: string) => emit('update:modelValue', val),
+});
+
+const inputRef = ref<HTMLElement | null>(null);
+const isEditing = ref<boolean>(!props.modelValue && !props.disabled);
+
+function enableEdit() {
+	if (props.disabled) return;
+	isEditing.value = true;
+}
+
+function disableEdit() {
+	isEditing.value = false;
+}
+
+onClickOutside(inputRef, () => {
+	if (!props.disabled) {
+		disableEdit();
+	}
+});
+
+onKeyStroke('Enter', (e) => {
+	if (isEditing.value && !props.disabled) {
+		e.preventDefault();
+		disableEdit();
+	}
+});
+</script>
+
+<template>
+	<div class="field focus-keyphrase-container">
+		<div class="field-input-group">
```

Comment:
The way this is styled it looks like it would be an array of tags, I would either recommend just having an input OR removing the gray background pill and just having the text... like the slug in the below image.
![image](https://github.com/user-attachments/assets/65d4012e-a0f7-4c33-931f-bb348d520e01)